plugin/src/main/java/net/dzikoysk/funnyguilds/config/migration/P0012_Migrate_old_broadcast_death_message_key.java

```diff
@@ -0,0 +1,24 @@
+package net.dzikoysk.funnyguilds.config.migration;
+
+import eu.okaeri.configs.migrate.builtin.NamedMigration;
+import net.dzikoysk.funnyguilds.config.PluginConfiguration;
+
+import static eu.okaeri.configs.migrate.ConfigMigrationDsl.move;
+
+public class P0012_Migrate_old_broadcast_death_message_key extends NamedMigration {
+
+        public P0012_Migrate_old_broadcast_death_message_key() {
+            super(
+                    "Migrate old boolean broadcast-death-message key to new enum death-message-receivers",
+                    move("broadcast-death-message", "death-message-receivers", oldValue -> {
+                        if (oldValue instanceof Boolean) {
+                            return (Boolean) oldValue
+                                    ? PluginConfiguration.DeathMessageReceivers.WORLD
```

Comment:
I think this case should also use `DeathMessageReceivers.ALL` to fix this bug for users with old configs